src/vs/workbench/browser/parts/quickopen/quickOpenController.ts

```diff
@@ -133,6 +133,10 @@ export class QuickOpenController extends Component implements IQuickOpenService
 		} else {
 			this.closeOnFocusLost = this.configurationService.getValue(CLOSE_ON_FOCUS_LOST_CONFIG);
 		}
+		if (this.quickOpenWidget) {
```

Comment:
Is this fixing a bug? (Or why is it now needed?)

Reply:
I dispose and set to null to enforce  a new creation of the quickOpenWidget to get it loaded with the changed configuration.

Reply:
See my last PR-commit.